### PR TITLE
fix: exclude cancelled requests as well

### PR DESCRIPTION
**Pull request by GaryPEGEOT**

### CODE REVIEW

File: es/index.mjs [hunk 1]
```diff
@@ -7,12 +7,17 @@ export const namespace = 'axios-retry';
...
-    isRetryAllowed(error)
-  ); // Prevents retrying unsafe errors
+	const CODE_EXCLUDE_LIST = [
+		'ERR_CANCELED',
+		'ECONNABORTED'
+	];
+
+	return (
+		!error.response &&
+		Boolean(error.code) && // Prevents retrying cancelled requests
```
Comment (mindhells) [inline es/index.mjs:17]: are they not excluded already with this condition?

Comment (GaryPEGEOT) [reply es/index.mjs:17]: Nope! Cancel error did retry

Comment (mindhells) [reply es/index.mjs:17]: sorry man, completely forgot about this
you can use the retryCondition to customise this but I think it makes sense by default to not retry on this error (which is triggered for example when using an abort controller)
merging it

Comment (mindhells) [reply es/index.mjs:17]: published as 3.8.0
thanks a lot @GaryPEGEOT

**Pull request merged**